documentation/logit-io.md

```diff
@@ -128,8 +128,41 @@ filter {
       # Debug: Comment this line to keep the original object
       remove_field => "[app][payload][params]"
     }
+     mutate {
```

Comment:
Formatting:
- Fix the indentation
- Add comments to explain the mutate, one for ruby and one for .NET